docs/literate/src/files/custom_semidiscretization.jl

```diff
@@ -0,0 +1,312 @@
+#src # Custom semidiscretizations
+
+# As described in the [overview section](@ref overview-semidiscretizations),
+# semidiscretizations are high-level descriptions of spatial discretizations
+# in Trixi.jl. Trixi.jl's main focus is on hyperbolic conservation
+# laws represented in a [`SemidiscretizationHyperbolic`](@ref).
+# Hyperbolic-parabolic problems based on the advection-diffusion equation or
+# the compressible Navier-Stokes equations can be represented in a
+# [`SemidiscretizationHyperbolicParabolic`](@ref). This is described in the
+# [basic tutorial on parabolic terms](@ref parabolic_terms) and its extension to
+# [custom parabolic terms](@ref adding_new_parabolic_terms).
+# In this tutorial, we will describe how these semidiscretizations work and how
+# they can be used to create custom semidiscretizations involving also other tasks.
+
+
+# ## Overview of the right-hand side evaluation
+
+# The semidiscretizations provided by Trixi.jl are set up to create `ODEProblem`s from
+# [SciML ecosystem for ordinary differential equations](https://diffeq.sciml.ai/latest/).
```

Comment:
```suggestion
# [SciML's ecosystem for ordinary differential equations](https://diffeq.sciml.ai/latest/).
```

Reply:
SciML is the abbreviation of "Scientific Machine Learning". Expanding SciML like this, SciML's ecosystem sounds weird to me.

Reply:
Should it then be
```suggestion
# the [SciML ecosystem for ordinary differential equations](https://diffeq.sciml.ai/latest/).
```
?

Reply:
Good point. I did that in 58db0589dd8b82b8e5f7dcf0520152cc69bb2d57.